civis/io/_tables.py

```diff
@@ -384,19 +384,43 @@ def civis_to_multifile_csv(sql, database, job_name=None, api_key=None,
     -------
     unload_manifest: dict
         A dictionary resembling an AWS manifest file. Has the following keys:
-        ``'header'``, ``'query'``, ``'entries'``, respresenting the columns
-        from the query, the query itself, and a list of dictionaries for each
-        unloaded CSV part, each containing its file ``'id'``, ``'name'``,
-        ``'size'``, and unsigned and signed S3 urls, ``'url'`` and
-        ``'url_signed'``, respectively.
+
+        'query': str
+            The query.
+
+        'header': list of str
+            The columns from the query.
+
+        'entries': list of dict
+            Each dict has the following keys:
+
+            'id': int
+                File ID.
+            'name': str
+                Filename
+            'size': int
+                File size
```

Comment:
Do you know what the units are? Is this bytes?

Reply:
Yes, it's bytes -- added just now.